src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -597,6 +597,25 @@ protected function whereInRaw(Builder $query, $where)
         return '0 = 1';
     }
 
+    /**
+     * Compile a "where fulltext" clause.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  array  $where
+     * @return string
+     */
+    public function whereFullText(Builder $query, $where)
+    {
+        $columns = collect($where['columns'])
+            ->map(fn ($column) => $this->wrap($column))
+            ->implode(' || ');
+
+        // Third parameter CONTAINS() function
+        $labelSearch = $where['options']['label'] ?? 0;
+
+        return "CONTAINS({$columns}, {$this->parameter($where['value'])}, {$labelSearch}) > 0";
```

Comment:
I think its possible to make an auto incrementing counter to make subsequent orWhereFullText not need any label option.

Reply:
Fixed! Thanks for idea!